WORKSPACE

```diff
@@ -16,11 +16,10 @@
 #
 workspace(name = "io_bazel_rules_webtesting")
 
-http_archive(
+git_repository(
```

Comment:
Please switch this back to http_archive. You can put the commit number in the locations where you see 0.3.2 in url and strip_prefix and you will need to update the sha256.